### PR TITLE
Update Readme and add parameter definitions

### PR DESCRIPTION
**Pull request by RitikJaiswal75**
Updated the readme and provided parameter annotations for verifying bot function.

Made the verify bot function consistent by just returning boolean values rather than different return types

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,3 +49,20 @@ In case you are interested in it you could read more about it here:
...
+DISCORD_TOKEN: The token generated for your bot while creating a discord application
+DISCORD_PUBLIC_KEY: Public key of your Discord bot helps to verify the bot and apply interaction url
+DISCORD_APPLICATION_ID: The application id of your bot.
+DISCORD_GUILD_ID: Id of the guild where you want to install the slash commands.
+```
+
+To add more commands you need to modify following files:
+
+```
+- src/constants/commands.ts (Export your commands a constant)
```
Comment (heyrandhir) [inline README.md:65]:
```suggestion
- src/constants/commands.ts (Export your commands as a constant)
```

File: README.md [hunk 2]
```diff
@@ -49,3 +49,20 @@ In case you are interested in it you could read more about it here:
...
+- src/register.ts (import the newly added command here and ass it to the commands array)
```
Comment (heyrandhir) [inline README.md:66]:
```suggestion
src/register.ts (import the newly added command here and assign it to the commands array)
```

File: src/utils/verifyBot.ts [hunk 3]
```diff
@@ -3,6 +3,13 @@ import { Env } from "../typeDefinitions/default.types";
 import JSONResponse from "./JsonResponse";
 import * as error from "../constants/errors";
 
+/**
+ *
+ * @param request { Request } : request that the worker receives
+ * @param env { Env }: the ctx (contest) which contains the secrets put in as wrangler secrets.
```
Comment (heyrandhir) [inline src/utils/verifyBot.ts:7]: can you confirm `ctx` means contest or context ?

File: src/utils/verifyBot.ts [hunk 4]
```diff
@@ -3,6 +3,13 @@ import { Env } from "../typeDefinitions/default.types";
 import JSONResponse from "./JsonResponse";
 import * as error from "../constants/errors";
 
+/**
+ *
+ * @param request { Request } : request that the worker receives
```
Comment (heyrandhir) [inline src/utils/verifyBot.ts:6]:
```suggestion
* @param request { Request } : request the worker receives
```

**Review by heyrandhir — CHANGES_REQUESTED** [review]

File: src/utils/verifyBot.ts (hunk 3 above)
Comment (RitikJaiswal75) [reply src/utils/verifyBot.ts:7]: yes

**Review by heyrandhir — APPROVED** [review]

**Review by rohan09-raj — APPROVED** [review]

**Pull request merged**